example/media/nasa.mp4

```diff
@@ -0,0 +1 @@
+Happy Thanksgiving From the International Space Station.mp4
```

Comment:
this isn't a mp4 file..

Reply:
This is a soft link to the mp4 file. I will replace 'nasa.mp4' with the actual file and remove the file named 'Happy Thanksgiving From the International Space Station.mp4'; the reference link will be given.

Reply:
Hi, I have updated the files, please review them again.

Reply:
how big is the mp4 file? can you make sure it's as small as possible?

Reply:
The size of nasa.mp4 has been reduced to 4.1MB.